### PR TITLE
#13084 - Linked SelfReporting message should be shown as side compone…

### LINKED ISSUE #13084: Linked SelfReporting message should be shown as side component in UI [2]
### Feature Description

Link the processed SelfReporting message to the according entity / entities and show in UI

### Added Value/Benefit

Link the processed SelfReporting message to the according entity / entities and show in UI

### Acceptance Criteria

- [ ]
- [ ]
- [ ]

### Implementation Details

_No response_

### Mockups

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by sergiupacurariu**
…nt in UI

Fixes #13084

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13107

File: sormas-backend/src/main/java/de/symeda/sormas/backend/selfreport/SelfReportFacadeEjb.java [hunk 1]
```diff
@@ -77,6 +78,8 @@ public class SelfReportFacadeEjb
 
 	@EJB
 	private LocationFacadeEjbLocal locationFacade;
+	@EJB
+	private SelfReportService selfReportService;
```
Comment (leventegal-she) [inline sormas-backend/src/main/java/de/symeda/sormas/backend/selfreport/SelfReportFacadeEjb.java:82]: SelfReportService is already injected, so this EJB is not needed

File: sormas-ui/src/main/java/de/symeda/sormas/ui/utils/BulkOperationHandler.java [hunk 2]
```diff
@@ -42,7 +42,7 @@ public class BulkOperationHandler<T extends HasUuid> {
 	/**
 	 * Amount of DTOs that have to be selected for the progress layout to be displayed.
 	 */
-	public static final int BULK_ACTION_PROGRESS_THRESHOLD = 40;
+	public static final int BULK_ACTION_PROGRESS_THRESHOLD = 3;
```
Comment (leventegal-she) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/utils/BulkOperationHandler.java:45]: revert this

File: sormas-api/src/main/resources/strings.properties [hunk 3]
```diff
@@ -672,6 +672,7 @@ headingSecurityAlert=Security Alert
 headingSelectCampaign = Select a campaign
 headingSetOutbreakStatus = Set status of all districts:
 headingShowExternalMessage = Message
+headingSelfReportSideComponent = Self reporting
```
Comment (leventegal-she) [inline sormas-api/src/main/resources/strings.properties:675]: please cahnge to `Self reports`

File: sormas-api/src/main/resources/captions.properties [hunk 4]
```diff
@@ -2962,6 +2962,7 @@ specailCaseAccessNoAccessGranted=No special access granted for this case
 SelfReport = Self report
 SelfReport.type = Type
 SelfReport.reportDate = Report date
+SelfReport.caseReference = Case Reference
```
Comment (leventegal-she) [inline sormas-api/src/main/resources/captions.properties:2965]: please change to "Case reference number"

File: sormas-api/src/main/java/de/symeda/sormas/api/selfreport/SelfReportListEntryDto.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+import de.symeda.sormas.api.utils.pseudonymization.PseudonymizableIndexDto;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class SelfReportListEntryDto extends PseudonymizableIndexDto implements Serializable {
+
+    public static final String I18N_PREFIX = "SelfReport";
+
+
```
Comment (leventegal-she) [inline sormas-api/src/main/java/de/symeda/sormas/api/selfreport/SelfReportListEntryDto.java:13]: unnecessary white spaces

**Review by leventegal-she — CHANGES_REQUESTED** [review]

File: sormas-backend/src/main/java/de/symeda/sormas/backend/selfreport/SelfReportFacadeEjb.java (hunk 1 above)
Comment (sergiupacurariu) [reply sormas-backend/src/main/java/de/symeda/sormas/backend/selfreport/SelfReportFacadeEjb.java:82]: removed

File: sormas-ui/src/main/java/de/symeda/sormas/ui/utils/BulkOperationHandler.java (hunk 2 above)
Comment (sergiupacurariu) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/utils/BulkOperationHandler.java:45]: reverted

File: sormas-api/src/main/resources/strings.properties (hunk 3 above)
Comment (sergiupacurariu) [reply sormas-api/src/main/resources/strings.properties:675]: changed

File: sormas-api/src/main/resources/captions.properties (hunk 4 above)
Comment (sergiupacurariu) [reply sormas-api/src/main/resources/captions.properties:2965]: changed

File: sormas-api/src/main/java/de/symeda/sormas/api/selfreport/SelfReportListEntryDto.java (hunk 5 above)
Comment (sergiupacurariu) [reply sormas-api/src/main/java/de/symeda/sormas/api/selfreport/SelfReportListEntryDto.java:13]: reformat the code

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13107

**Review by leventegal-she — APPROVED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13107

**Pull request merged**